snippets/base/templates/base/includes/snippet_as.js

```diff
@@ -421,7 +421,9 @@ function sendMetric(metric, callback, href) {
       }
       return;
     {% else %}
-      if (!SNIPPET_METRICS_URL || (Math.random() > SNIPPET_METRICS_SAMPLE_RATE) || (!metric)) {
+
+    if (gSnippetsMap.get('telemetryEnabled') !== true || !SNIPPET_METRICS_URL ||
```

Comment:
Any particular reason you used `gSnippetsMap.get('telemetryEnabled') !== true` and not `!gSnippetsMap.get('telemetryEnabled')`?

I'm fine either way, this is pretty simple.